docs/concepts/build-backend.md

```diff
@@ -44,7 +44,7 @@ build-backend = "uv_build"
 To create a new project that uses the uv build backend, use `uv init`:
 
 ```console
-$ uv init
+$ uv init --lib
```

Comment:
Thank you!

I think we probably want `--package` here. Though the case could be made for either?

Reply:
Didn't notice there is also `--package`. A bit nuanced difference between package or library in my opinion. I see one comes out with `project.scripts` section. Fact that I can combine `--lib` with `--package` makes it a bit more confusing even...  

I'll leave it up to you. Definitely bare `init` was not enough but which one you'd like to go with I don't know.